docs/section23/Rule23-3.md

```diff
@@ -22,33 +22,46 @@
 
-- For each zone in B_RMR: `for zone_b in B_RMR...zones:`
+- For each zone in B_RMD: `for zone_b in B_RMD...zones:`
 
   - For each terminal in zone: `for terminal_b in zone_b.terminals:`
 
     - Get HVAC system serving terminal: `hvac_b = terminal_b.served_by_heating_ventilating_air_conditioning_systems`
   
       - Check if HVAC system is one of the eligible systems: `if hvac_b.id in eligible_hvac_system_ids:`
-
-        **Rule Assertion:**
-
-        - Case 1: For each terminal that is served by HVAC system that is Type-5, 7, 7a, 5b, 7b, 7c, 6, 8, 8a, 6b, 8b, if minimum volume setpoint is equal to 30% of peak primary design airflow or the rate required for minimum outside air, whichever is larger: `if terminal_b.minimum_airflow == MAX(terminal_b.primary_airflow * 0.3, terminal_b.minimum_outdoor_airflow): PASS`
-
-        - Case 2: Else, minimum volume setpoint is not equal to 30% of peak primary design airflow or the rate required for minimum outside air, whichever is larger: `else: FAIL`
+        
+        - get the minimum volume flow rate of the proposed terminal for the same zone.  This is to do the 3rd check - that the minimum airflow is greater than the rate required to comply with local standards.  It is assumed that the proposed model meets the minimum requirements of local standards.  First get the zone in the proposed model: `zone_p = get_component_by_id(P_RMD, Zone, zone_b.id)`
+        - create a variable to hold the total minimum volume flow rate of all terminals in the proposed zone.  This will be an array of values, one for each hour of the year.  We will sum the minimum volume flowrate for each terminal by the availability of the terminal to take into account cases where the proposed zone has multiple terminals that do not operate at the same time: `minimum_volume_list_p = []`
+        - look at each terminal in the proposed zone: `for terminal_p in zone_p.terminals:`
+          - get the HVAC system associated with the terminal: `hvac_p = terminal_p.served_by_heating_ventilating_air_conditioning_system`
+          - get the fan system: `fan_system_p = hvac_p.fan_system`
+          - get the operation schedule hourly values: `operation_schedule_hourly_values_p = fan_system_p.operating_schedule.hourly_values`
+          - get the terminal minimum volume flowrate: `min_volume_p = terminal_p.minimum_airflow`
+          - multiply each value in the hourly schedule by the minimum airflow rate, and append this to the list of minimum volume flowrates: `minimum_volume_list_p.append([i * min_volume_p for i in operation_schedule_hourly_values_p]`
+        - sum all of the values for each hour, so that the resulting list is a list of the sum of minimum flowrates across all terminals each hour of the year: `min_volume_flow_tot_p = [sum(values) for values in zip(*minimum_volume_list_p)]`
+        - find the minimum non-zero value in the list: `min_volume_p = min(filter(lambda x: x != 0, min_volume_flow_tot_p))`
```

Comment:
These two lines are calculating the minimum air volume for the HVAC system, not terminal. The assertion cases are not using HVAC system level data to confirm the calculation.

Reply:
@KarenWGard